File: 1791.find-center-of-star-graph.py
#
# @lc app=leetcode id=1791 lang=python
#
# [1791] Find Center of Star Graph
#

# @lc code=start
class Solution(object):
    def findCenter(self, edges):
        """
        :type edges: List[List[int]]
        :rtype: int
        """
        a, b = edges[0]
        c, d = edges[1]

        if a == c or a == d:
            return a
        return b
        
# @lc code=end

File: 1791.find-center-of-star-graph.java
/*
 * @lc app=leetcode id=1791 lang=java
 *
 * [1791] Find Center of Star Graph
 */

// @lc code=start
class Solution {
    public int findCenter(int[][] edges) {
        int a = edges[0][0];
        int b = edges[0][1];
        int c = edges[1][0];
        int d = edges[1][1];

        if (a == c || a == d) {
            return a;
        }
        return b;
    }
}
// @lc code=end

